index.test.js

```diff
@@ -755,7 +755,7 @@ describe('Deploy to ECS', () => {
             cluster: 'cluster-789',
             service: 'service-456',
             taskDefinition: 'task:def:arn',
-            forceNewDeployment: true
+            forceNewDeployment: false
```

Comment:
this condition is not correct - this test is meant to test the _success_ of passing in `forceNewDeployment`, so we would expect `ecs:update-service` to be called with this param set as `true`. If this is not occurring, either the setup of the test is wrong or the change is not behaving as we expect.